app/components/tweet/schema.yml

```diff
@@ -0,0 +1,33 @@
+_description: |
+  Embeds a tweet via the tweet’s URL.
+  _Similar to: [clay-facebook-post](https://github.com/nymag/sites/blob/master/components/clay-facebook-post/README.md), [clay-instagram](https://github.com/nymag/sites/blob/master/components/clay-instagram/README.md), [clay-pinterest](https://github.com/nymag/sites/blob/master/components/clay-pinterest/README.md), [tumblr-post](https://github.com/nymag/sites/blob/master/components/tumblr-post/README.md)_
```

Comment:
Since we don't have none of these components, we can omit this in the description